help/destinations/catalog/email-marketing/salesforce-marketing-cloud-exact-target.md

```diff
@@ -201,7 +201,7 @@ To validate that you have correctly set up the destination, follow the steps bel
 1. Monitor the segment summary and ensure that the count of profiles corresponds to the count created within the segment.
 ![Platform UI screenshot example showing Segment.](../../assets/catalog/email-marketing/salesforce-marketing-cloud-exact-target/segment.png)
 
-1. Log in to the Salesforce Marketing Cloud website. Then navigate to the **[!DNL Audience Builder]** > **[!DNL Contact Builder]** > **[!DNL All contacts]** > **[!DNL Email]** page and check if the profiles from the segment have been added.
+1. Log in to the [!DNL Salesforce Marketing Cloud](https://mc.exacttarget.com/) website. Then navigate to the **[!DNL Audience Builder]** > **[!DNL Contact Builder]** > **[!DNL All contacts]** > **[!DNL Email]** page and check if the profiles from the segment have been added.
```

Comment:
Looks like a missing double-bracket issue here. Remember, we need one set of square brackets for the DNL tag and another set of square brackets for the hyperlink. Thanks!

`[!DNL Salesforce Marketing Cloud](https://mc.exacttarget.com/)`

Reply:
All good as long as we double-check these instances going forward. Thanks for all your help!